src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -223,14 +223,18 @@ class WorkflowViewContent extends Component {
 
   renderSummary(invalidIO) {
     const { workspace: { canCompute } } = this.props
-    const { modifiedConfig, savedConfig, entityMetadata, saving, saved, activeTab } = this.state
+    const { modifiedConfig, savedConfig, entityMetadata, saving, saved, activeTab, errors } = this.state
     const { name, methodRepoMethod: { methodPath, methodVersion }, rootEntityType } = modifiedConfig
     const modified = !_.isEqual(modifiedConfig, savedConfig)
 
     const noLaunchReason = Utils.cond(
       [invalidIO.inputs || invalidIO.outputs, () => 'Add your inputs and outputs to Launch Analysis'],
```

Comment:
This line isn't right. Instead of `invalidIO.inputs`, we need `!_.isEmpty(errors.inputs)`, and same for outputs.